iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/compaction/IcebergCompactionService.java

```diff
@@ -53,7 +53,7 @@ public Boolean compact(Table table, CompactionInfo ci) throws Exception {
     CompactorUtil.checkInterrupt(CLASS_NAME);
 
     org.apache.iceberg.Table icebergTable = IcebergTableUtil.getTable(conf, table);
-    if (!IcebergCompactionEvaluator.isEligibleForCompaction(icebergTable, ci.partName, ci.type, conf)) {
+    if (!IcebergCompactionEvaluator.isEligibleForCompaction(icebergTable, ci, conf)) {
```

Comment:
could we rename to just `CompactionEvaluator`?

Reply:
Done